crates/providers/src/new.rs

```diff
@@ -70,6 +82,8 @@ impl<N, T> RootProviderInner<N, T> {
     }
 }
 
+// todo: adjust docs
+// todo: reorder
```

Comment:
reminder to open issue -- 

really like remco's thing here https://github.com/recmo/uint/blob/main/.github/scripts/issue_tracker.py & https://github.com/recmo/uint/blob/main/.github/workflows/issue_tracker.yml

Reply:
And we don't :D https://discord.com/channels/900503503162724452/1049944297799168080/1212754890338865224

Reply:
https://github.com/alloy-rs/alloy/issues/269